### ISSUE TITLE
L032 rule cannot be excluded by configuration

### ISSUE THREAD

**Issue by KulykDmytro** [sqlfluff/sqlfluff#1968]
```
select
    ch.dt,
    ch.created_on,
    player_id,
    ch.currency,
    ch.version,
    ch.reason,
    ch.source,
    ch.source_transaction_id,
    lb.balance_name
from changes as ch
left join last_balances as lb using (player_id)
```

## Expected Behaviour
when setting up `L032` in configuration: `exclude_rules`
sqlfluff lint should skip this warning

## Observed Behaviour
while set in configuration
```
L:  93 | P:  23 | L032 | Found USING statement. Expected only ON statements.
```
however inline `--noqa: L032` works as designed (skipping rule)

## Dialect
postgres

## Version
0.8.2 + Python 3.9.8

## Configuration
```
[sqlfluff]
output_line_length = 180
exclude_rules = L011,L012,L022,L031,L032,L034,L042
dialect = postgres
```

**Comment by tunetheweb** [comment]
I can't repeat this. Are you sure your `.sqlfluff` file is being picked up correctly?

**Comment by jpy-git** [comment]
@KulykDmytro, I've checked too and also don't see this issue 

![image](https://user-images.githubusercontent.com/80432516/143612739-dfd5ea96-b3af-4bb3-a6c9-b388e694d948.png)

**Comment by KulykDmytro** [comment]
sorry for misleading you - there is a typo in query
should be like this at L10:
```sql
    lb.balance_name
```

**Comment by jpy-git** [comment]
@KulykDmytro No worries, thanks for updating! 😄
